app/controllers/stealth/controller/replies.rb

```diff
@@ -76,7 +76,10 @@ def send_replies
           Stealth.trigger_reply(flow, state, current_message)
         end
 
-        def say(reply, thread_id: nil)
+        def say(reply, **args)
+          thread_id = args[:thread_id]
+          delete_message = args[:delete_message]
```

Comment:
In the interest of making our life easier when we add additional drivers/service clients, let's start organizing the service args into a private method.

Something like:

```
        def service_args(**args)
          case service_client
            when 'slack'
            return {              
              thread_id: args.fetch(:thread_id, nil),
              delete_message: args.fetch(:delete_message, nil)
            }
          end
        end
```

Additionally, we should use fetch to make sure we're not erroring out if we're not passing the `delete_message` arg which I believe is optional?